### ISSUE TITLE
Retrieving unknown variant via GET returns AssertionError

### ISSUE THREAD

**Issue by rvidal** [Network-of-BioThings/myvariant.py#1]
Just tested the python wrapper and noticed that when querying for a single variant that is not in any database, a 404 response is returned, thus breaking the 200 assertion in the `_get` method.

```python
import myvariant
mv = myvariant.MyVariantInfo()
mv.getvariant('chr17:c.3748G>T') # unknown variant
```

If I attempt to use `mv.getvariants('chr17:c.3748G>T')` (which uses POST), I get a response like so:
```python
In [7]: mv.getvariants(["chr7:g.140453133Tmv.getvariants('chr17:c.3748G>T')
querying 1-1...done.
Out[7]: [{u'notfound': True, u'query': u'chr17:c.3748G>T'}]
```

The API actually returns 404 on this request, thus makes the assert break on line 195 of `__init__.py`
So, while the python script should account for these scenarios (tests should account for these situations too), I think it might be something the API should be doing differently.

**Comment by newgene** [comment]
Thanks for your feedback. I agree, the more friendly return from an unknown variant could be just a "None" instead of raising exception. Will make that change shortly.

**Comment by newgene** [comment]
Fixed in 6cfff54, now return None from ```getvariant``` method when the input id does not exist.

**Comment by rvidal** [comment]
Great simple fix. The API still returns the 404 (which makes sense) but the python wrapper handles it in a more "friendly" manner.
